src/services/database/dynamodb/__tests__/dynamo-ehr-transfer-tracker.integration.test.js

```diff
@@ -1,13 +1,15 @@
 import { EhrTransferTracker } from '../dynamo-ehr-transfer-tracker';
 import { v4 as uuid } from 'uuid';
-import { RecordType } from '../../../constants/enums';
-import { cleanupRecordsForTest } from '../../../utilities/integration-test-utilities';
+import { RecordType } from '../../../../constants/enums';
+import {
+  createInboundRecordForTest,
+} from '../../../../utilities/integration-test-utilities';
 
 describe('EhrTransferTracker', () => {
   const testConversationId = uuid();
 
   afterEach(async () => {
-    await cleanupRecordsForTest(testConversationId);
+    // await cleanupRecordsForTest(testConversationId);
```

Comment:
Is this supposed to be commented out?

Reply:
Thanks. Will push a commit to revert it back